mikeio1d/various.py

```diff
@@ -99,3 +103,24 @@ def allow_nested_autocompletion_for_ipython(cls: Type):
     from IPython.core import guarded_eval
 
     guarded_eval.EVALUATION_POLICIES["limited"].allowed_getattr.add(cls)
+
+
+def open(file_name: str | Path, **kwargs) -> Res1D | Xns11:
+    """Open a file type supported by MIKE IO 1D file."""
+    # import here to avoid circular imports
+    from mikeio1d.res1d import Res1D
```

Comment:
OK, I see a problem here :-). Actually, it looks that various.py is not a good place for it, so I would just elevate the open method to a open.py.

Reply:
Yep not the best. I'll put it in `top_level.py` for now, then we can revisit the naming of that module 😄